packages/exmg-button/demo/button-demo-styles.scss

```diff
@@ -13,4 +13,9 @@ div {
 }
 div > * {
   padding: 8px;
-}
\ No newline at end of file
+}
+
+exmg-button.progress-button {
```

Comment:
Remove class and use `exmg-button[progress]` instead

Reply:
Done